### ISSUE TITLE
Training From Scratch

### ISSUE THREAD

**Issue by sunnnnnnnny** [facebookresearch/demucs#215]
## ❓ Questions
Hello, I am now training from scratch with musdb data. How many epochs will I train for a reasonable result?

**Comment by sunnnnnnnny** [comment]
Hello, I want to mix musdb and some of my own data to train the model from scratch. The dataloader and dataset of musdb are included in the project. Does it mean that I only need to build a dataloader and dataset of my own dataset, and use this for each epoch Two dataloaders are used to train the model (without data enhancement), and the validation set is unchanged. At present, the loss of the first 20 epoch validation set of the model is very slow to converge. What will cause the problem?
